<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Генерация .mif файлов</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>371</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Генерация файла на основе полученных данных</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>324</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Выбор файла</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>исходный тип данных</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="START_BIN">
           <property name="text">
            <string>BIN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="START_OCT">
           <property name="text">
            <string>OCT</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="START_DEC">
           <property name="text">
            <string>DEC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="START_HEX">
           <property name="text">
            <string>HEX</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Открыть файл</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="OPEN"/>
         </item>
         <item>
          <widget class="QPushButton" name="OPEN_ALL">
           <property name="text">
            <string>Обзор...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Сохранить файл</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SAVE"/>
         </item>
         <item>
          <widget class="QPushButton" name="SAVE_ALL">
           <property name="text">
            <string>Обзор...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="GET">
         <property name="text">
          <string>Получить файл</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>241</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Настройка избранного файла</string>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>220</width>
         <height>107</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Тип данных</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_BIN">
          <property name="text">
           <string>BIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_OCT">
          <property name="text">
           <string>OCT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_DEC">
          <property name="text">
           <string>DEC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_HEX">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>122</y>
         <width>220</width>
         <height>107</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Тип адреса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_BIN">
          <property name="text">
           <string>BIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_OCT">
          <property name="text">
           <string>OCT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_DEC">
          <property name="text">
           <string>DEC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_HEX">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Кол-во бит</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="BIT"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Кол-во данных</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RADIX"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="APPLY">
          <property name="text">
           <string>Принять</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLineEdit" name="INFORMATION">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>324</width>
        <height>90</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Генерация файла на основе мат. формул</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>241</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>Настройка файла</string>
      </property>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>220</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Кол-во бит</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="BIT_GENERATION"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Кол-во данных</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RADIX_GENERATION"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="APPLY_GENERATION">
          <property name="text">
           <string>Принять</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>107</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Тип данных</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_GENERATION_BIN">
          <property name="text">
           <string>BIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_GENERATION_OCT">
          <property name="text">
           <string>OCT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_GENERATION_DEC">
          <property name="text">
           <string>DEC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="DATA_GENERATION_HEX">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>122</y>
         <width>221</width>
         <height>107</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Тип адреса</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_GENERATION_BIN">
          <property name="text">
           <string>BIN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_GENERATION_OCT">
          <property name="text">
           <string>OCT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_GENERATION_DEC">
          <property name="text">
           <string>DEC</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="ADDRES_GENERATION_HEX">
          <property name="text">
           <string>HEX</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLineEdit" name="INFORMATION_GENERATION">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>270</y>
        <width>328</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>328</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>Генерация файла</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Сохранить файл</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SAVE_GENERATION"/>
           </item>
           <item>
            <widget class="QPushButton" name="SAVE_GENERATION_ALL">
             <property name="text">
              <string>Обзор...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QRadioButton" name="SUBTRACTION">
             <property name="text">
              <string>ВЫЧИТАНИЕ</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="DIVISION">
             <property name="text">
              <string>ДЕЛЕНИЕ</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="MULTIPLICATION">
             <property name="text">
              <string>УМНОЖЕНИЕ</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QRadioButton" name="RADICAL">
             <property name="text">
              <string>КОРЕНЬ</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="DEGREE">
             <property name="text">
              <string>СТЕПЕНЬ</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QRadioButton" name="ADDITION">
             <property name="text">
              <string>СЛОЖЕНИЕ</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QRadioButton" name="COSINE">
             <property name="text">
              <string>КОСИНУС</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QRadioButton" name="SINUS">
             <property name="text">
              <string>СИНУС</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QRadioButton" name="ZERO">
             <property name="text">
              <string>НУЛЕВОЙ</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Сколько чисел после запятой</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="NUMBER_2"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Число</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="NUMBER"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>От какого числа(не включительно)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="MIN_RADIX"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>До какого числа(включительно)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="MAX_RADIX"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="GET_GENERATION">
           <property name="text">
            <string>Сгенерировать собственный файл</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
